src/components/application_manager/include/application_manager/policies/policy_retry_sequence.h

```diff
@@ -42,7 +42,8 @@ class PolicyHandler;
 class RetrySequence : public threads::ThreadDelegate {
  public:
   explicit RetrySequence(PolicyHandler* const policy_handler);
-  void threadMain();
+  void threadMain() OVERRIDE;
+  void exitThreadMain() OVERRIDE;
 
  private:
   PolicyHandler* const policy_handler_;
```

Comment:
@AByzhynar I do not know. This is just an interface without implementation created by @justinjdickow I decided just to do some fixes in respect to proper thread stopping.